portal-impl/src/portal.properties

```diff
@@ -7910,6 +7911,14 @@
     #
     module.framework.portal.dir=${module.framework.base.dir}/portal
 
+    #
+    # Set the directory where the module framework portal WAR applications will
+    # be placed.
+    #
+    # Env: LIFERAY_MODULE_PERIOD_FRAMEWORK_PERIOD_PORTAL_MINUS_WAR_PERIOD_DIR
+    #
+    module.framework.portal-war.dir=${module.framework.base.dir}/portal-war
```

Comment:
@marianoalvarosaiz Did you test your change? I do not think it works.

Your have a typo here. `${module.framework.portal.war.dir}` and `module.framework.portal-war.dir`